spec/services/location_data_service_spec.rb

```diff
@@ -185,6 +245,45 @@
       new_location = FactoryBot.create(:delivery_location)
       expect(new_location.id).to eq 41
     end
+
+    describe "new recap locations" do
+      before do
+        LocationDataService.delete_existing_and_repopulate
+        @location_engineer_pt = Locations::HoldingLocation.find_by(code: 'engineer$pt')
```

Comment:
rubocop doesn't like these; you could just define them directly in the `it` block.

Reply:
there are 2 it blocks that uses them

Reply:
doesn't the 2nd one just use the one variable? I think it's fine to define it again. Or you can do `let`s like rubocop says. whichever you prefer!

Reply:
yeah it's only the one in the second it.